src/components/editions/series.tsx

```diff
@@ -0,0 +1,36 @@
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { css } from '@emotion/core';
+import { titlepiece } from '@guardian/src-foundations/typography';
+import { news } from '@guardian/src-foundations/palette';
+import { remSpace } from '@guardian/src-foundations';
+
+import { Item } from 'item';
+import { maybeRender } from 'lib';
+
+
+// ----- Component ----- //
+
+const styles = css`
+    ${titlepiece.small()}
+    color: ${news[400]};
+    font-size: 17px;
```

Comment:
No rems?

Reply:
Good point. Sorry, this is a temporary override of the fonts allowed in Source (cc @SiAdcock) because Titlepiece 17 is not currently allowed. However, you're right, it should be in `rem`s, I'll fix that.